test/axes/timeAxisTests.ts

```diff
@@ -130,6 +130,20 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("end ticks' lengths equal to endTickLength() when tierLabelPosition is set to center", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    axis.tierLabelPositions(["center", "center"]);
+    scale.domain([new Date("2010-01-01"), new Date("2014-01-01")]);
+    axis.renderTo(svg);
+    let endTicks = axis.content().selectAll(`.${Plottable.Axis.END_TICK_MARK_CLASS}`);
+    assert.operator(endTicks.size(), ">=", 1, "At least one end tick mark is selected in the test");
+    endTicks.each(function(d, i){
+      let tickLength = Math.abs(+this.getAttribute("y1") - +this.getAttribute("y2"));
+      assert.strictEqual(tickLength, axis.endTickLength(), "end tick marks's length should equal to endTickLength()");
```

Comment:
We're comparing pixels to our own values.  Let's use `assert.closeTo` to not have this test fail due to pixel diffing reasons.